guides/source/active_record_validations.md

```diff
@@ -103,18 +109,19 @@ irb> p.new_record?
 => false
 ```
 
-Creating and saving a new record will send an SQL `INSERT` operation to the
-database. Updating an existing record will send an SQL `UPDATE` operation
-instead. Validations are typically run before these commands are sent to the
+Saving a new record will send an SQL `INSERT` operation to the
+database, whereas updating an existing record will send an SQL `UPDATE`
+operation. Validations are typically run before these commands are sent to the
 database. If any validations fail, the object will be marked as invalid and
-Active Record will not perform the `INSERT` or `UPDATE` operation. This avoids
-storing an invalid object in the database. You can choose to have specific
-validations run when an object is created, saved, or updated.
+Active Record will not perform the `INSERT` or `UPDATE` operation. This helps
+to avoid storing an invalid object in the database. You can choose to have
+specific validations run when an object is created, saved, or updated.
 
-CAUTION: There are many ways to change the state of an object in the database.
-Some methods will trigger validations, but some will not. This means that it's
-possible to save an object in the database in an invalid state if you aren't
-careful.
+WARNING: While validations usually prevent invalid data from being saved to the
+database, it's important to be aware that not all methods in Rails trigger
+validations. Some methods allow changes to be made directly to the database
+without performing validations. As a result, if you're not careful, it’s
+possible to bypass validations and save an object in an invalid state.
```

Comment:
Would it make sense to link to the "skipping validations" section below here to serve as an example of which methods do this?